ansible/roles/ansible-datamover-api/tasks/get_python.yml

```diff
@@ -0,0 +1,26 @@
+---
+- name: find location when using python3
+  shell: which python3
+  register: python3
+  when: PYTHON_VERSION=="python3"
+
+- name: find location when using python3
+  shell: which python
+  register: python2
+  when: PYTHON_VERSION=="python2"
+
+- debug:
```

Comment:
Do not display debug msg by default , you can use verbosity level to 2 , so that it can only  disply in debug mode

Reply:
requested changes done